src/main/scala/play/socketio/SocketIOEventAck.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2017-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+package play.socketio
+
+import akka.util.ByteString
+import play.api.libs.json.JsValue
+import scala.util.Either
+
+/**
+ * A socket.io ack function.
+ */
+trait SocketIOEventAck {
```

Comment:
Move SocketIOEventAck from the Java API as we're not supporting Scala 2.11 here.